pkg/kubemark/hollow_kubelet.go

```diff
@@ -111,7 +110,6 @@ func NewHollowKubelet(
 		HostUtil:                  hostutil.NewFakeHostUtil(nil),
 		PodStartupLatencyTracker:  kubeletutil.NewPodStartupLatencyTracker(),
 		NodeStartupLatencyTracker: kubeletutil.NewNodeStartupLatencyTracker(),
-		TracerProvider:            noopoteltrace.NewTracerProvider(),
```

Comment:
do we have to remove this?

Reply:
yes, but we need add a global TracerProvider.